### DISCUSSION TITLE
How do I opt into enabling fulltext Search for my posts on Mastodon

### DISCUSSION THREAD

**Discussion by suramyatomar** [Automattic/wordpress-activitypub#469] [Q&A] [answered]
As you might be aware the new version of Mastodon (4.2.0) brings full text search, so you can search by words instead of hashtags.  However, for privacy reasons this only includes posts by people who have actively opted into full text search.

How do I enable this for my Blog which is posting to Mastodon via the activitypub? 

- Suramya

**Comment by pfefferle** [comment, marked as answer]
This is already enabled by default if you use v1.0.1.

**Reply by suramyatomar** [reply]
Awesome. Thanks for the confirmation :)

**Reply by pfefferle** [reply]
you're welcome :)
